### ISSUE TITLE
[Question] How are the candidate answers in VQA datasets used?

### ISSUE THREAD

**Issue by jihuishan** [haotian-liu/LLaVA#1283]
### Question

As OKVQA, A-OKVQA, VQAv2, .etc, contain ten candidate answers. So how are these answers used in training? Are all candidate answers involved? Or, only the main answer is used? Thank you.

**Comment by zealousChinese** [comment]
Does the OKVQA dataset have main answers? Which attribute in the annotation indicates the main answer?

**Comment by jihuishan** [comment]
> swer?

Ops, it doesn't. I usually regard the most common answer as the main one.

**Comment by jihuishan** [comment]
Yet the main concern here is how the answers are used. A traditional way I know is to compute the loss by comparing the generated text towards each candidate answer and then average or sum the results.

**Comment by zealousChinese** [comment]
> > Yet the main concern here is how the answers are used. A traditional way I know is to compute the loss by comparing the generated text towards each candidate answer and then average or sum the results.
> 
> Which probably not the llava way here, as me observed.

Yes, I didn't find the performance of LLaVA on OK-VQA in the paper. I also want to use LLaVA for the OK-VQA dataset.
